### DISCUSSION TITLE
Can I change the link of the Perchance project so I can use my own styles?

### DISCUSSION THREAD

**Discussion by AstroCatPlays** [manh9011/Perchance-T2I-Desktop#1] [Q&A] [answered]
Is it possible to change the link of the Perchance project so I can use my own styles? If so, how?

**Comment by manh9011** [comment, marked as answer]
Currently, there is no way to integrate styles from Perchance web to Perchance desktop. But if you have some programming experience, you can change the styles in file `Perchance/ArtStyle.cs`  to add new styles.

Example:
```
["Casual Photo"] = new(
    prompt: input => $"{input.Description}, casual photo",
    negative: input => $"{input.Negative}, bad photo, bad lighting, high production value, unnatural studio lighting, commercial photoshoot, photoshopped, terrible photo, disfigured"
),
```

**Reply by AstroCatPlays** [reply]
Oh alright, thanks!
